app/ui-sidenav/client/roomList.js

```diff
@@ -115,7 +115,6 @@ Template.roomList.helpers({
 
 		return !['unread', 'f'].includes(group.identifier) || (rooms.length || (rooms.count && rooms.count()));
 	},
-
```

Comment:
This change should be removed

Reply:
https://github.com/assistify/Rocket.Chat/pull/621